src/Symfony/Component/Security/Core/Tests/Encoder/Argon2iPasswordEncoder.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Tests\Encoder;
+
+use Symfony\Component\Security\Core\Encoder\Argon2iPasswordEncoder;
+
+/**
+ * @author Elnur Abdurrakhimov <elnur@elnur.pro>
+ */
+class Argon2iPasswordEncoderTest extends \PHPUnit_Framework_TestCase
+{
+    const PASSWORD = 'password';
+
+    public function setUp()
+    {
+        if (PHP_VERSION_ID < 70200 && !extension_loaded('libsodium')) {
```

Comment:
Yes, absolutely. I did change from version-checking to feature-checking in `Argon2iPasswordEncoder` but I unfortunately either forgot to update it here, or it snuck in whilst rebasing (either way I missed it and shall update when I get home from work).

The `isPasswordValid()` method checks for the function that it uses, but neither `encodePassword()` or `isPasswordValid()` check for the existence of `sodium_memzero()` - is this something that should be done in case one of the check functions has been defined in user-land code? It's a pretty far off edge-case but should at least be aknowledged...